R/Estonia.R

```diff
@@ -0,0 +1,86 @@
+#' Estonia Class for downloading, cleaning and processing notification data
+#' @description Information for downloading, cleaning
+#'  and processing COVID-19 region data for Estonia
+#'
+# nolint start
+#' @source \url{https://www.terviseamet.ee/et/koroonaviirus/avaandmed}
+# nolint end
+#' @export
+#' @concept dataset
+#' @family subnational
+#' @examples
+#' \dontrun{
+#' region <- Estonia$new(verbose = TRUE, steps = TRUE, get = TRUE)
+#' region$return()
+#' }
+Estonia <- R6::R6Class("Estonia",
+  inherit = DataClass,
+  public = list(
+
+    # Core Attributes
+    #' @field origin name of origin to fetch data for
+    origin = "Estonia",
+    #' @field supported_levels A list of supported levels.
+    supported_levels = list("1"),
+    #' @field supported_region_names A list of region names in order of level.
+    supported_region_names = list("1" = "county"),
+    #' @field supported_region_codes A list of region codes in order of level.
+    supported_region_codes = list("1" = "iso_3166_2"),
+    #' @field common_data_urls List of named links to raw data.
+    # nolint start
+    common_data_urls = list(
+      "main" = "https://opendata.digilugu.ee/opendata_covid19_test_county_all.csv" # nolint
```

Comment:
two different data sources?

Reply:
One is the link to the (Estonian language) landing page which presents the data and explains the content. The other is the direct URL for downloading the CSV. But I think there’s some nolint overkill here.

Reply:
Ah right got you. In that case all good.